### ISSUE TITLE
LVM2 plugin - creation of vg fails with the value '/dev/XXX' doesn't match the pattern 

### ISSUE THREAD

**Issue by aheinzel** [openmediavault/openmediavault#307]
I can't create a new LVM2 volume group on a fresh install of OMV V 4.1.3-1.

I installed the LVM2 plugin and added /dev/vdb as physical volume. Creation of a new volume group with the before added physical volume as member fails with the following error message
```
devices: The value '/dev/vdb' doesn't match the pattern '^(.+[,;])*.+$'.

Error #0:
OMV\Json\SchemaValidationException: devices: The value '/dev/vdb' doesn't match the pattern '^(.+[,;])*.+$'. in /usr/share/php/openmediavault/json/schema.inc:478
Stack trace:
#0 /usr/share/php/openmediavault/json/schema.inc(370): OMV\Json\Schema->checkPattern('/dev/vdb', Array, 'devices')
#1 /usr/share/php/openmediavault/json/schema.inc(297): OMV\Json\Schema->validateString('/dev/vdb', Array, 'devices')
#2 /usr/share/php/openmediavault/json/schema.inc(624): OMV\Json\Schema->validateType('/dev/vdb', Array, 'devices')
#3 /usr/share/php/openmediavault/json/schema.inc(395): OMV\Json\Schema->checkProperties(Object(stdClass), Array, '')
#4 /usr/share/php/openmediavault/json/schema.inc(285): OMV\Json\Schema->validateObject(Object(stdClass), Array, '')
#5 /usr/share/php/openmediavault/json/schema.inc(257): OMV\Json\Schema->validateType(Object(stdClass), Array, '')
#6 /usr/share/php/openmediavault/rpc/paramsvalidator.inc(59): OMV\Json\Schema->validate(Object(stdClass))
#7 /usr/share/php/openmediavault/rpc/serviceabstract.inc(178): OMV\Rpc\ParamsValidator->validate('{"name":"vg1","...')
#8 /usr/share/openmediavault/engined/rpc/logicalvolumemgmt.inc(381): OMV\Rpc\ServiceAbstract->validateMethodParams(Array, 'rpc.logicalvolu...')
#9 [internal function]: OMVRpcServiceLogicalVolumeMgmt->createVolumeGroup(Array, Array)
#10 /usr/share/php/openmediavault/rpc/serviceabstract.inc(123): call_user_func_array(Array, Array)
#11 /usr/share/php/openmediavault/rpc/rpc.inc(86): OMV\Rpc\ServiceAbstract->callMethod('createVolumeGro...', Array, Array)
#12 /usr/sbin/omv-engined(536): OMV\Rpc\Rpc::call('LogicalVolumeMg...', 'createVolumeGro...', Array, Array, 1)
#13 {main}
```

**Comment by votdev** [comment]
What type of hardware and locale are you using?

The symptom you're talking about seems to be the same as some users mentioned when they are using the openmediavault-nut plugin.

It seems that there are some problems with regular expressions and PHP. Maybe introduced by the installed locale. If you know regex, then you should see that there is absolutely no issue with the regex and the value to be checked:

regex: ^(.+[,;])*.+$
value: /dev/vdb

Nothing really difficult here, but PHP returns an invalid regex. Currently i've no idea how to fix that because this issue is not reproducable using german and english locale.

**Comment by votdev** [comment]
Can you please upgrade openmediavault core to the latest version? This is because the error message will then include an error code which is returned by PHP and this might help to track down the issue.

**Comment by aheinzel** [comment]
I upgraded to the latest version using omv's update management and the error is now gone - sorry. I must mention I also installed all other available updates...

I initially installed the latest release iso image (openmediavault_4.1.3-amd64.iso) in a KVM virtual machine for testing purposes. Locale: en_US.UTF-8

If you are interested I can create another VM, install again from the iso, do not upgrade to the latest version, see if the error is reproducible and then selectively install updates?
